src/test/java/org/torusresearch/torusutilstest/keys/key.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN EC PRIVATE KEY-----
+MEECAQAwEwYHKoZIzj0CAQYIKoZIzj0DAQcEJzAlAgEBBCCD7oLrcKae+jVZPGx52Cb/lKhdKxpXjl9eGNa1MlY57A==
```

Comment:
don't commit this

Reply:
Normally I would not have.

I am not against reverting this, however in this instance it is very well known. Like to the extent that it may as well be here. It appears in multiple of our codebases as well as in some of our example documentation.

Is there another reason you would like it removed?

Reply:
it's used to generate id tokens for our test verifiers. not a good practice to commit this

Reply:
Have removed this.